api/Element.json

```diff
@@ -5917,7 +5917,7 @@
               }
             ],
             "ie": {
-              "version_added": null
+              "version_added": false
```

Comment:
I thought this (api.Element. onfullscreenchange) might be wrong, but https://software.hixie.ch/utilities/js/live-dom-viewer/saved/8475 shows that actually only `document.onmsfullscreenchange` was in IE11, not `element.onmsfullscreenchange`. But I found and fixed another problem: https://github.com/mdn/browser-compat-data/pull/6684